### PR TITLE
fixed broken install in docker file

### PR DESCRIPTION
**Pull request by maorgoldberg**
Fixed issue that caused curl version to break the docker file by updating its version.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,6 +1,6 @@
 FROM garethr/kubeval:0.15.0
 
-RUN apk --no-cache add curl==7.67.0-r1 bash==5.0.11-r1 git==2.24.3-r0 openssh-client==8.1_p1-r0
+RUN apk --no-cache add curl==7.67.0-r2 bash==5.0.11-r1 git==2.24.3-r0 openssh-client==8.1_p1-r0
```
Comment (stefanprodan) [inline Dockerfile]: Can you please remove the version pinning for all packages

**Review by stefanprodan — CHANGES_REQUESTED** [review]

**Comment by maorgoldberg** [comment]
Removing version pinning causes checks to fail

**Comment by stefanprodan** [comment]
That lint rule is kinda silly, we need to remove it.

**Comment by maorgoldberg** [comment]
Yea I'll take a look at the linting action and add an ignore for it and create a separate pr for it. However this one should fix dockerfile errors without issue regardless of the failed lint

**Review by stefanprodan — APPROVED** [review]
LGTM

Thanks @maorgoldberg

**Comment by stevenpall** [comment]
Could we please get a new release cut that includes this fix? We try not to use `@master`.

**Comment by maorgoldberg** [comment]
> Could we please get a new release cut that includes this fix? We try not to use `@master`.

Is an action needed on my part?

**Comment by stevenpall** [comment]
> > Could we please get a new release cut that includes this fix? We try not to use `@master`.
> 
> Is an action needed on my part?

Nope, all good now. The `v3.5.0` release includes this fix. Thanks!

**Pull request merged**